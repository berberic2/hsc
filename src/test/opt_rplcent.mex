test/simple.hsc (7,7): Note 46: replaced `\-4;' by `&uuml;'
test/simple.hsc (7,9): Note 46: replaced `\-33;' by `&szlig;'
test/simple.hsc (7,29): Note 46: replaced `\-10;' by `&ouml;'
test/simple.hsc (7,40): Note 46: replaced `\-36;' by `&Uuml;'
test/simple.hsc (7,43): Note 46: replaced `\-28;' by `&auml;'
test/simple.hsc (7,46): Note 46: replaced `\-28;' by `&auml;'
