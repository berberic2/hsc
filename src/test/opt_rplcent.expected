<HTML><HEAD><TITLE>Simple test</TITLE></HEAD>
<BODY>just a &quot;simple" test with no special
features &amp; &lt;tags&gt;;.

And now something German:

Nur f&uuml;r&szlig; Te&szlig;ten der d&ouml;&ouml;fen &Uuml;ml&auml;ut&auml;.

</BODY></HTML>
